kolibri/core/content/utils/content_request.py

```diff
@@ -202,16 +280,33 @@ def incomplete_downloads_queryset():
     Returns a queryset used to determine the incomplete downloads, with and without metadata, as
     well as the total import size if it does have metadata
     """
-    return (
+    qs = (
         ContentDownloadRequest.objects.filter(status__in=INCOMPLETE_STATUSES)
         .order_by("requested_at")
         .annotate(
             has_metadata=Exists(
                 ContentNode.objects.filter(pk=OuterRef("contentnode_id"))
             ),
             total_size=_total_size_annotation(),
+            is_learner_download=Case(
+                When(
+                    source_model=FacilityUser.morango_model_name,
+                    then=FixedExists(
```

Comment:
Is the error this is fixing specific to using the Exists subquery in a `then` statement?